free-programming-books.md

```diff
@@ -493,6 +493,7 @@ Original Source: [List of freely available programming books](http://web.archive
 * [SVG Essentials](http://commons.oreilly.com/wiki/index.php/SVG_Essentials)
 * [The Quest for Artificial Intelligence: A History of Ideas and Achievements](http://ai.stanford.edu/~nilsson/QAI/qai.pdf) - Nils J. Nilsson (PDF)
 * [The Z Notation: A Reference Manual, Second Edition](http://spivey.oriel.ox.ac.uk/~mike/zrm/zrm.pdf) - J. M. Spivey (PDF)
+* [Varoius:  A different free book every day](http://www.packtpub.com) - Various authors (PDF, ePub, Mobi, Kindle)
```

Comment:
Thanks for your PR!

* Please replace *Varoius* with *Packt* and reorder accordingly
* Please remove *different* (it is not useful)
* Please replace *http* with *https*
* Please add  `*(account required)*` at the end of the line.